spec/rails_multitenant_spec.rb

```diff
@@ -0,0 +1,53 @@
+include RailsMultitenant
```

Comment:
Why are we including this module here?

Reply:
@jturkel I had copied some of the other specs in this gem, e.g.
https://github.com/salsify/rails-multitenant/blob/master/spec/global_context_registry_spec.rb

Given your question though, I added `extend self` into the top-level `RailsMultitenant` module. Prior to that, it was necessary to `include RailsMultitenant` to make the delegated methods visible. Using `extend self` avoids that and matches [what `GlobalContextRegistry` does](https://github.com/salsify/rails-multitenant/blob/master/lib/rails_multitenant/global_context_registry.rb#L10).

Reply:
Sounds good. I'm not sure why `global_context_registry_spec.rb` has the `include RailsMultitenant`. It looks like it was done to just avoid typing `RailsMultitenant::`.